src/utils/processActionReturnData.ts

```
@@ -0,0 +1,13 @@
+import cloneDeep from "lodash.clonedeep";
+import { convertDecimalToNumber } from "@/utils/convertDecimalToNumbers";
+import { removeFields } from "@/utils/query.utils";
+
+export function processActionReturnData<T>(data: T | T[], limitFields: string[] = []) {
+  if (data instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)